3 pages more options fixed

Nppatel97 opened the pull request:
This PR addresses the fix for the bug "more options not working on certain break-point". Currently it is addressed on Agenda, My Profile and My Tutors pages. More fixes are on the way. Some design fixes are also made like notifications pane now shows inside the viewport in windowed browser.

sdevalapurkar commented:
This PR addresses this: https://github.com/sdevalapurkar/schedulearn/issues/105

sdevalapurkar commented on schedule_lessons/dashboard/static/dashboard/css/agenda.css:
```diff
@@ -281,15 +283,48 @@ html body {
+}
+
 @media(min-width: 576px) {
+
```
nit: remove newline

Nppatel97 replied:
About the code quality: We need to do some major work on the code in all the css files. There are unwanted, duplicate and unnecessary classes, things are sort of hard to find and overall a little messy. I'll take a thorough look at the code and we'll open a new PR if necessary.

sdevalapurkar replied:
sounds great man! would really appreciate that

sdevalapurkar commented on schedule_lessons/dashboard/static/dashboard/js/agenda.js:
```diff
@@ -10,6 +10,10 @@ $(document).ready(function () {
     // $('#options_btn').toggleClass('addBtnAnim');
   });
 
+  $("#secondNavBtn").click(function(){
```
nit/consistency note: I personally prefer single quotes in javascript code but depending on how we have done it everywhere else i am ok with double quotes as long as we are consistent :) 
also, just check this out for fun if you'd like: https://bytearcher.com/articles/single-or-double-quotes-strings-javascript/

Nppatel97 replied:
I'll make it consistent. Thanks for noticing

sdevalapurkar commented on schedule_lessons/dashboard/static/dashboard/js/relationships.js:
```diff
@@ -1,6 +1,10 @@
   $('[data-toggle="tooltip"]').tooltip();
 
+  $("#secondNavBtn").click(function(){
+    $("#navbarSupportedContent").slideToggle("fast");
```
question: what does passing in 'fast' do? I know it is the duration of the toggle right?

Nppatel97 replied:
Yeah, it toggles the menu faster. It matches the speed of the Bootstrap navbar.

sdevalapurkar replied:
cool, good to know!

Nppatel97 commented:
Alright, I think I have corrected all the pages that had more options button. They should all work now. Also fixed the navbar on schedule_lesson page so now it is consistent with rest of UI. Some other minor adjustments to the notifications panel were also made for mobile versions.

sdevalapurkar commented on schedule_lessons/dashboard/static/dashboard/js/my_profile.js:
```diff
@@ -89,6 +89,10 @@ $(document).ready(function () {
     $("#reviewTab").fadeOut(300);
   });
 
+  $("#secondNavBtn").click(function(){
```
lets use single quotes here and in line below

sdevalapurkar commented on schedule_lessons/dashboard/static/dashboard/js/relationships.js:
```diff
@@ -1,6 +1,10 @@
 $(document).ready(function () {
   $('[data-toggle="tooltip"]').tooltip();
 
+  $("#secondNavBtn").click(function(){
```
single quotes

Nppatel97 replied:
Done. Merging the PR
